src/page-components/CartSummary/CartSummary.tsx

```diff
@@ -0,0 +1,174 @@
+"use client";
+import React from "react";
+import { Button, styled } from "@mitodl/smoot-design";
+import { Typography } from "@mui/material";
+import { createTheme } from "@mitodl/smoot-design";
+import { InputBase as Input } from "@mui/material";
+import { Card } from "../../components/Card/Card";
+import StyledCard from "../../components/Card/StyledCard";
+import CartSummaryItem, {
+  CartSummaryItemContainer,
+  CartSummaryItemTitle,
+  CartSummaryItemValue,
+} from "../CartSummaryItem/CartSummaryItem";
+import PlaceOrderButton from "../PlaceOrderButton/PlaceOrderButton";
+
+import {
+  usePaymentsBasketRetrieve,
+  usePaymentsBasketAddDiscount,
+} from "@/services/ecommerce/payments/hooks";
+import { UseQueryResult } from "@tanstack/react-query";
+import { BasketWithProduct } from "@/services/ecommerce/generated/v0";
+
+type CartSummaryProps = {
+  cartId: number;
+};
+
+type CartSummaryDiscountProps = {
+  systemSlug: string;
+};
+
+const theme = createTheme();
+
+const CartSummaryContainer = styled.div(() => ({
+  width: "488px",
+  padding: "0",
+  ["> div"]: {
+    padding: "32px",
+  },
+}));
+
+const CartSummaryReceiptContainer = styled.div(() => ({
+  borderBottom: "1px solid #DDE1E6",
+  padding: "8px 0",
+  margin: "8px 0",
+  width: "100%",
+}));
+
+const CartSummaryTotalContainer = styled.div`
+  ${{ ...theme.typography.h5 }},
```

Comment:
Comma here (and a few other places) should be semi-colon if you're doing the styled.div`...css..`. Can also do `styled.div({ ...object...})` then its a comma, since js object.

I would avoid creating a theme (`const theme = createTheme()`) for this. It's possible it's different from the actual theme that's passed into `ThemeProvider` in providers.tsx

You _could_ import the theme from `src/components/theme`. But I'd just do:
```tsx
const CartSummaryTotalContainer = styled.div`
  ${({ theme }) => ({ ...theme.typography.h5 })};
```
Or lately I've started preferring the object syntax:
```tsx
const CartSummaryTotalContainer = styled.div(({theme}) => {
   ...theme.typography.h5,
  marginBottom: 20px,
  marginTop: 8px
})
```
because i feel it makes accessing the theme a little easier.